config/routes.rb

```diff
@@ -1,6 +1,10 @@
 FloodRiskEngine::Engine.routes.draw do
+  resources :enrollments
+
```

Comment:
`resources :enrollments` already defined below.
maybe change the one below to `resources :enrollments, only: [:new, :create] do`